client/me/help/help-contact/index.jsx

```diff
@@ -119,7 +119,7 @@ const HelpContact = React.createClass( {
 
 		notifications.forEach( olarkActions.sendNotificationToOperator );
 
-		analytics.tracks.recordEvent( 'calypso_help_live_chat_begin' );
+		analytics.tracks.recordEvent( 'calypso_help_live_chat_begin', { site_plan: site.plan.product_id } );
```

Comment:
Since this is a `connect`ed component, it might be nice to utilize the [`getSitePlan`](https://github.com/Automattic/wp-calypso/blob/master/client/state/sites/selectors.js#L452) selector and pass in `product_id` as a `prop` to this component